src/main/webapp/index.html

```diff
@@ -8,13 +8,38 @@
   </head>
   <body>
     <header>
-      <h1>smallcity</h1>
+      <h1 class="logo">smallcity</h1>
+      <div class="zipcode-container">
+        your zipcode: 
+      </div>
     </header>
 
     <div class="main-container">
       <div class="map-container">
       </div>
       <div class="results-container">
+        <div class="results-card">
+          <div class="results-image">
+            image
```

Comment:
Should this say "image"?

Reply:
I just had it there as placeholder text so everyone knew what it was. Will be removed once we add in the js functionality

Reply:
Gotcha. Then please prefer a comment, that should serve the same purpose, over placeholder text that will appear in the dom.